client/scss/components/_userbar.scss

```
@@ -261,6 +257,66 @@ $positions: (
   }
 }
 
+.w-dialog--userbar {
+  // Display off to the side of the page rather than in the middle.
+  inset-inline-start: auto;
+
+  .w-dialog__close-button {
+    $size: theme('spacing.6');
+    width: $size;
+    height: $size;
+    top: calc(-1 * $size / 2);
+    inset-inline-end: calc(-1 * $size / 2);
+    border-radius: theme('borderRadius.full');
+    border: 2px solid theme('colors.primary.DEFAULT');
+    background-color: theme('colors.white.DEFAULT');
+  }
+
+  .w-dialog__content {
+    padding: 0;
+  }
+
+  .w-dialog__title {
+    @apply w-h3;
+    padding: theme('spacing.4') theme('spacing.5');
+  }
+
+  .w-dialog__subtitle {
+    @apply w-body-text;
+    padding-inline-start: theme('spacing.5');
+  }
+
+  .w-a11y-result__row {
+    border-top: 1px solid theme('colors.grey.100');
+  }
+
+  .w-a11y-result__header {
+    margin: 0;
+    padding: theme('spacing.4') theme('spacing.5');
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    font: inherit;
+  }
+
+  .w-a11y-result__name {
+    color: theme('colors.primary.200');
+    font-weight: theme('fontWeight.bold');
+  }
+
+  .w-a11y-result__container {
+    padding: 0 theme('spacing.5') theme('spacing.5') theme('spacing.5');
+  }
+
+  .w-a11y-error-selector {
+    background: theme('colors.grey.50');
+    color: theme('colors.grey.600');
+    border-radius: theme('borderRadius.DEFAULT');
+    margin-right: 10px;
```

Our [linter in CI](https://app.circleci.com/pipelines/github/wagtail/wagtail/12184/workflows/f5d64d31-2444-4346-9bfb-2b939936775b/jobs/35004) is complaining with:

```
client/scss/components/_userbar.scss
 315:5  ✖  Unexpected property "margin-right"  property-disallowed-list
```

Needs to be:

```suggestion
    margin-inline-start: 10px;
```